parm/archive/gdas.yaml.j2

```diff
@@ -0,0 +1,132 @@
+        {% endif %}
+        {% if DO_JEDISNOWDA %}
+        - "{{ COM_SNOW_ANALYSIS | relpath(ROTDIR) }}/{{ head }}snowstat.tgz"
+        {% endif %}
+        {% endif %}  # Full cycle
+        - "logs/{{ cycle_YMDH }}/{{ RUN }}fcst.log"
+        # TODO explicitly name the atmos_prod log files to archive
+        - "logs/{{ cycle_YMDH }}/{{ RUN }}atmos_prod_f*.log"
+        {% for fhr in range(0, FHMAX + FHOUT, FHOUT) %}
+        - "{{ COM_ATMOS_GRIB_0p25 | relpath(ROTDIR) }}/{{ head }}pgrb2.0p25.f{{ '%03d' % fhr }}"
+        - "{{ COM_ATMOS_GRIB_0p25 | relpath(ROTDIR) }}/{{ head }}pgrb2.0p25.f{{ '%03d' % fhr }}.idx"
+        - "{{ COM_ATMOS_GRIB_1p00 | relpath(ROTDIR) }}/{{ head }}pgrb2.1p00.f{{ '%03d' % fhr }}"
+        - "{{ COM_ATMOS_GRIB_1p00 | relpath(ROTDIR) }}/{{ head }}pgrb2.1p00.f{{ '%03d' % fhr }}.idx"
+        - "{{ COM_ATMOS_HISTORY | relpath(ROTDIR) }}/{{ head }}atm.logf{{ '%03d' % fhr }}.txt"
+        - "{{ COM_ATMOS_HISTORY | relpath(ROTDIR) }}/{{ head }}atmf{{ '%03d' % fhr }}.nc"
+        - "{{ COM_ATMOS_HISTORY | relpath(ROTDIR) }}/{{ head }}sfcf{{ '%03d' % fhr }}.nc"
+        - "{{ COM_ATMOS_MASTER | relpath(ROTDIR) }}/{{ head }}sfluxgrbf{{ '%03d' % fhr }}.grib2"
+        - "{{ COM_ATMOS_MASTER | relpath(ROTDIR) }}/{{ head }}sfluxgrbf{{ '%03d' % fhr }}.grib2.idx"
+        {% endfor %}
+    optional:
+        {% if current_cycle != SDATE and MODE == "cycled" %}
+        {% if not DO_JEDIATMVAR %}
+        - "{{ COM_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}gsistat"
+        {% endif %}
+        {% if DO_VERFRAD %}
+        - "{{ COM_ATMOS_RADMON | relpath(ROTDIR) }}/bad_diag.{{ cycle_YMDH }}"
+        - "{{ COM_ATMOS_RADMON | relpath(ROTDIR) }}/bad_pen.{{ cycle_YMDH }}"
+        - "{{ COM_ATMOS_RADMON | relpath(ROTDIR) }}/low_count.{{ cycle_YMDH }}"
+        - "{{ COM_ATMOS_RADMON | relpath(ROTDIR) }}/warning.{{ cycle_YMDH }}"
+        {% endif %}
+        {% if DO_VERFOZN %}
+        {% set oznmon_types = [
+           "gome_metop-b", "omi_aura", "ompslp_npp", "ompsnp_n20",
+           "ompsnp_npp", "ompstc8_n20", "ompstc8_npp", "sbuv2_n19"
+        ] %}
+        {% for group in [ "horiz", "time" ] %}
+        {% if group == "horiz" %} {% set suffix = ".gz" %} {% else %} {% set suffix = "" %} {% endif %}
+        {% for type in oznmon_types %}
+        - "{{ COM_ATMOS_OZNMON | relpath(ROTDIR) }}/{{ group }}/{{ type }}.anl.ctl"
+        - "{{ COM_ATMOS_OZNMON | relpath(ROTDIR) }}/{{ group }}/{{ type }}.anl.{{ cycle_YMDH }}.ieee_d{{ suffix }}"
+        - "{{ COM_ATMOS_OZNMON | relpath(ROTDIR) }}/{{ group }}/{{ type }}.ges.ctl"
+        - "{{ COM_ATMOS_OZNMON | relpath(ROTDIR) }}/{{ group }}/{{ type }}.ges.{{ cycle_YMDH }}.ieee_d{{ suffix }}"
+        {% endfor %}
+        {% endfor %}
+        {% endif %}
+        {% endif %}
+        {% for fhr in range(FHMIN, FHMAX + FHOUT, FHOUT) %}
+        - "{{ COM_ATMOS_MASTER | relpath(ROTDIR) }}/{{ head }}sfluxgrbf{{ '%03d' % fhr }}.grib2"
```

Comment:
The sflux files are saved earlier in this yaml (Lines 95/96)

Reply:
Yep, I'll delete these from optional.